tests/infra_test.go

```diff
@@ -175,6 +175,47 @@ var _ = Describe("Infrastructure", func() {
 			metricsURL = fmt.Sprintf("https://%s:%d/metrics", pod.Status.PodIP, 8443)
 		})
 
+		It("should find one leading virt-controller and two ready", func() {
```

Comment:
@rmohr I know that a better approach would be to add a unit-test that checks that the controller actually increments/decrements the corresponding metric with a mock but it would require writing something similar to the framework we have for virt-operator but for virt-controller... I think that we should do it in the future, but it will require much more work and I don't think it should be done in this PR.

Reply:
Would you mind open an issue so that we don't forget the task?

Reply:
Done.